src/main/java/frc/team4909/robot/subsystems/intake/IntakeSubsystem.java

```diff
@@ -34,13 +34,25 @@ public void hatchPanelIntakeClose() {
         hatchPanelSolenoid.set(DoubleSolenoid.Value.kReverse);
     }
 
-    public void setCargoIntakeSpeed(double speed) {
-        speed = -speed;
+    public void holdCargoIntake(){
         if(hasCargo()){
-            speed = 0.0;
+            setCargoIntakeSpeed(RobotConstants.cargoIntakeHoldSpeed);
+        } else {
+            setCargoIntakeSpeed(0);
+        }
+    }
+
+    public void setCargoIntakeSpeed(double speed) {
+        if(hasCargo() && speed > 0){
+            speed = RobotConstants.cargoIntakeHoldSpeed;
         }
-        //System.out.println(getCargoIntakeCurrent());
-        cargoIntakeMotor.set(speed);
+
+        cargoIntakeMotor.set(-speed);
+    }
+
+    @Override
+    public void periodic() {
+        System.out.println(hasCargo());
```

Comment:
uhhh this might want to be removed and/or given more description for the comp code because right now, this is cluttering our logs with true/false